### ISSUE TITLE
cheat command does not work on Windows 7

### ISSUE THREAD

**Issue by ematta** [defunkt/cheat#10]
When running the command `cheat rdebug` I get the following issue:

```
C:/Ruby192/lib/ruby/gems/1.9.1/gems/cheat-1.3.0/lib/cheat.rb:215:in `fork': fork() function is unimplemented on this machine (NotImplementedError)
        from C:/Ruby192/lib/ruby/gems/1.9.1/gems/cheat-1.3.0/lib/cheat.rb:215:in  `run_pager'
        from C:/Ruby192/lib/ruby/gems/1.9.1/gems/cheat-1.3.0/lib/cheat.rb:129:in  `show'
        from C:/Ruby192/lib/ruby/gems/1.9.1/gems/cheat-1.3.0/lib/cheat.rb:35:in `block in fetch_sheet'
        from C:/Ruby192/lib/ruby/1.9.1/open-uri.rb:150:in `open_uri'
        from C:/Ruby192/lib/ruby/1.9.1/open-uri.rb:671:in `open'
        from C:/Ruby192/lib/ruby/1.9.1/open-uri.rb:33:in `open'
        from C:/Ruby192/lib/ruby/gems/1.9.1/gems/cheat-1.3.0/lib/cheat.rb:31:in `fetch_sheet'
        from C:/Ruby192/lib/ruby/gems/1.9.1/gems/cheat-1.3.0/lib/cheat.rb:27:in `sheets'
        from C:/Ruby192/lib/ruby/gems/1.9.1/gems/cheat-1.3.0/bin/cheat:10:in `<top (required)>'
        from C:/Ruby192/bin/cheat:19:in `load'
        from C:/Ruby192/bin/cheat:19:in `<main>'
```

**Comment by sferik** [comment]
I merged this commit, which ought to fix the problem: https://github.com/defunkt/cheat/commit/ac119a7c181103026727c650284e243460117679

Can you please verify this fix on version 1.3.2, which I just released?

**Comment by ematta** [comment]
Sorry for the long delay. I tested this today and got the following:

```
C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:346:in `open_http': 404 Not Found  (OpenUR
I::HTTPError)
        from C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:775:in `buffer_open'
        from C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:203:in `block in open_loop'
        from C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:201:in `catch'
        from C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:201:in `open_loop'
        from C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:146:in `open_uri'
        from C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:677:in `open'
        from C:/Ruby193/lib/ruby/1.9.1/open-uri.rb:33:in `open'
        from C:/Ruby193/lib/ruby/gems/1.9.1/gems/cheat-1.3.3/lib/cheat.rb:31:in
`sheets'
        from C:/Ruby193/lib/ruby/gems/1.9.1/gems/cheat-1.3.3/bin/cheat:10:in `<t
op (required)>'
        from C:/Ruby193/bin/cheat:23:in `load'
        from C:/Ruby193/bin/cheat:23:in `<main>'
```
